README.md

```diff
@@ -231,7 +231,7 @@ CFLAGS to disable the compilation optimization parameters:
 ```shell
 $ export CFLAGS="-g -O0"
 $ ./build.sh
-$ ./configure
+$ ./configure --enable-assertions=yes
```

Comment:
Please explain the behavior of this new option. First it's enough to place that in a single comment.

Later we should consider to create a separated documentation / extend the existing one with all options.

Reply:
https://github.com/owasp-modsecurity/ModSecurity/pull/3104/commits/2c8684eeb0e99b25a4d488d802910cba927153bb